docs/kyma/docs/04-05-aks-installation.md

```diff
@@ -140,91 +149,97 @@ Delegate the management of your domain to Azure DNS. Follow these steps:
     cat kyma-config-cluster.yaml | sed -e "s/__PROXY_EXCLUDE_IP_RANGES__/10.0.0.1/g" | sed -e "s/__DOMAIN__/$SUB_DOMAIN.$DNS_DOMAIN/g" |sed -e "s/__TLS_CERT__/$TLS_CERT/g" | sed -e "s/__TLS_KEY__/$TLS_KEY/g"|sed -e "s/__.*__//g"  >my-kyma.yaml
     ```
 
-5. The output of this operation is the `my_kyma.yaml` file. Use it to deploy Kyma on your AKS cluster.
+3. The output of this operation is the `my_kyma.yaml` file. Use it to deploy Kyma on your AKS cluster.
 
 
 ### Using your own image
 
-1. Checkout [kyma-project](https://github.com/kyma-project/kyma) and enter the root folder.
-
-2. Build an image that is based on the current Installer image and includes the current installation and resources charts. Run:
+1. Build an image that is based on the current Installer image and includes the current installation and resources charts. Run:
 
     ```
     docker build -t kyma-installer:latest -f tools/kyma-installer/kyma.Dockerfile . --build-arg INSTALLER_VERSION=63484523
     ```
 
-3. Push the image to your Docker Hub:
+2. Push the image to your Docker Hub:
     ```
     docker tag kyma-installer:latest {YOUR_DOCKER_LOGIN}/kyma-installer:latest
     ```
     ```
     docker push {YOUR_DOCKER_LOGIN}/kyma-installer:latest
     ```
 
-4. Prepare the deployment file:
+3. Prepare the deployment file:
 
     ```
     cat installation/resources/installer.yaml <(echo -e "\n---") installation/resources/installer-config-cluster.yaml.tpl  <(echo -e "\n---") installation/resources/installer-cr-cluster.yaml.tpl | sed -e "s/__PROXY_EXCLUDE_IP_RANGES__/10.0.0.1/g" | sed -e "s/__DOMAIN__/$SUB_DOMAIN.$DNS_DOMAIN/g" |sed -e "s/__TLS_CERT__/$TLS_CERT/g" | sed -e "s/__TLS_KEY__/$TLS_KEY/g" | sed -e "s/__.*__//g" > my-kyma.yaml
     ```
 
-5. The output of this operation is the `my_kyma.yaml` file. Modify it to fetch the proper image with the changes you made (`{YOUR_DOCKER_LOGIN}/kyma-installer:latest`). Use the modified file to deploy Kyma on your AKS cluster.
-
+4. In the `my-kyma.yaml` file created in the previous step, change the image URL to `{YOUR_DOCKER_LOGIN}/kyma-installer:latest`.
+    ```
+    kind: Deployment
+    metadata:
+      name: kyma-installer
+      namespace: kyma-installer
+      ......
+        image: eu.gcr.io/kyma-project/develop/installer:30bf314d
+    ```
+    Use the modified file to deploy Kyma on your AKS cluster.
 
 ## Deploy Kyma
 
-1. Configure kubectl to use your new cluster and deploy Kyma Installer with your configuration. Run:
-    ```
-    az aks get-credentials --resource-group $RS_GROUP --name $CLUSTER_NAME
-    ```
-2. Deploy Kyma using the `my-kyma` custom configuration file you created. Run:
+1. Deploy Kyma using the `my-kyma` custom configuration file you created. Run:
     ```
     kubectl apply -f my-kyma.yaml
     ```
-3. Check if the Pods of Tiller and the Kyma Installer are running:
+    >**NOTE:** In case you receive an error: `Error from server (MethodNotAllowed)` run the command again before going to point no 2.
```

Comment:
```suggestion
    >**NOTE:** In case you receive the `Error from server (MethodNotAllowed)` error, run the command again before going to step 2.
```